### DISCUSSION TITLE
Question on glob pattern with exclude folder

### DISCUSSION THREAD

**Discussion by dreamworlds1234** [AnWeber/httpyac#639] [General]
Hi, I am trying to run the httpyac.js with glob pattern as following:

`node /httpyac/bin/httpyac.js send --bail --all --output short --output-failed short --timeout 10000 !special/**/*.http --env "test1"`

Here is the folder structure:

test1 \
  case1.http \
  case2.http \
  case3.http \
test2 \
  case1.http \
  case2.http \
test3 \
  case1.http \
  case2.http \
special \
  case1.http \
  case2.http \

I want to run all the httpyac tests under the current folder, EXCEPT *.http under the special folder, but it doesn't seem to work.

From the official README file, it says the fileName can be blob pattern.  It works when we use **/*.http, but not the exclude glob pattern.

Arguments:
  fileName                  path to file or glob pattern

Please help.

**Comment by AnWeber** [comment]
I use [globby](https://github.com/AnWeber/httpyac/blob/main/src/cli/send/send.ts#L194) under the hood which supports negated.patterns. Maybe I need to change the way I call globby. I will takena look

**Comment by dreamworlds1234** [comment]
Hi @AnWeber , I think I found the solution, I need to use the glob pattern like **/!(special)/*.http

Thanks!
